addOns/automation/src/main/java/org/zaproxy/addon/automation/jobs/JobUtils.java

```diff
@@ -17,6 +17,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/**
+ * This file contains different helper functions to perform tasks like object mapping.
+ * */
```

Comment:
This should be just before the class declaration and discuss the class (not file).